### PR TITLE
IA-609 active instruction

### PR DESCRIPTION
**Pull request by syoung-smallwisdom**
I've decided to tackle the heart rate step first so the active instruction is the first part of that.  

Requires: https://github.com/Sage-Bionetworks/ResearchSuite-AppleOS/pull/6

### CODE REVIEW

File: CRFModuleValidation/Assets.xcassets/12MinFullScreen.imageset/Contents.json [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
       "filename" : "12MinFullScreen.pdf"
     }
-  ]
+  ],
+  "info" : {
+    "version" : 1,
+    "author" : "xcode"
+  },
+  "properties" : {
+    "preserves-vector-representation" : true
```
Comment (Erin-Mounts) [inline CRFModuleValidation/Assets.xcassets/12MinFullScreen.imageset/Contents.json:13]: yay, finally!

File: CRFModuleValidation/Assets.xcassets/Colors/dodgerBlue.colorset/Contents.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    "version" : 1,
+    "author" : "xcode"
+  },
+  "colors" : [
+    {
+      "idiom" : "universal",
+      "color" : {
+        "color-space" : "srgb",
+        "components" : {
+          "red" : "62",
```
Comment (Erin-Mounts) [inline CRFModuleValidation/Assets.xcassets/Colors/dodgerBlue.colorset/Contents.json:12]: this is weird--some of the "srgb"-colorspace colors use integers from 0 to 255 for their red/green/blue components, some (e.g. macaroniAndCheese) use a float between 0.000 and 1.000. Do they all seem to work? If so, I guess the asset compiler decides what to do with them based on whether or not they have a decimal point?

**Review by Erin-Mounts — APPROVED** [review]
Couple of random comments, looks good

File: CRFModuleValidation/Assets.xcassets/Colors/dodgerBlue.colorset/Contents.json (hunk 2 above)
Comment (syoung-smallwisdom) [reply CRFModuleValidation/Assets.xcassets/Colors/dodgerBlue.colorset/Contents.json:12]: Not sure. I defined them all using 0-255.  The color name thing is iOS 11 only and I just used the Zeplin names for consistency.

**Pull request merged**